celesta-sql/src/main/java/ru/curs/celesta/score/Column.java

```diff
@@ -116,7 +118,7 @@ public final void delete() throws ParseException {
      *
      * @return
      */
-    public abstract Object getDefaultValue();
+    public abstract V getDefaultValue();
```

Comment:
Generic parameter works well here except for ``BinaryColumn`` type (see my comment above).

Reply:
...and that's because Binary Column cannot be used in `setRange`, so it's OK)